src/sentry/workflow_engine/endpoints/validators/base/data_condition.py

```diff
@@ -26,12 +26,18 @@ def validate(self, attrs):
         return attrs
 
 
+class BaseDataConditionGroupValidator(CamelSnakeSerializer):
+    logic_type = serializers.CharField(required=True)
+    organization_id = serializers.IntegerField(required=True)
+    conditions = BaseDataConditionValidator(many=True)
+
+
 class NumericComparisonConditionValidator(BaseDataConditionValidator):
     comparison = serializers.FloatField(
         required=True,
         help_text="Comparison value to be compared against value from data.",
     )
-    result = serializers.ChoiceField(
+    condition_result = serializers.ChoiceField(
```

Comment:
I changed the variable names to match what the payload calls them mostly because later on I can just do `...create(**data)` without having to rename everything